stylesheets/blue/components/_nav.scss

```diff
@@ -249,5 +258,10 @@ $Nav-item-color-hover: $Theme-color-yellowOrange;
   .Nav-menu {
     display: flex;
     align-items: center;
+
+    padding-right: $Theme-spacing-small;
+    padding-left: $Theme-spacing-small;
+    margin-right: -$Theme-spacing-small;
+    margin-left: -$Theme-spacing-small;
```

Comment:
What's wrong with negative margins?
This was the most effortless way of doing this.
Basically I needed it to have more space to the sides. The problem is, in the right side there's the Nav's edge, so it would push the menu button to the left. That's why I'm nullifying that with the negative margin.
For the right side of the menu button, I guess I don't need to nullify anything, I was just being consistent.

I'm not sure how your suggestion works